client/blocks/plan-storage/style.scss

```
@@ -1,6 +1,8 @@
 .plan-storage {
 	display: flex;
 	color: var(--studio-gray-80);
```

```suggestion
	color: inherit;
```
Side note-ish, but `:hover` and `:visited` selectors don't inherit from here, they inherit from the parent in the DOM. It's better to be consistent and change the default color rule to also inherit.